cmd/build/v2/build_test.go

```diff
@@ -370,13 +358,10 @@ func Test_getAccessibleVolumeMounts(t *testing.T) {
 	assert.Len(t, volumes, 1)
 }
 
-func createDockerfile() (string, error) {
-	dir, err := os.MkdirTemp("", "build")
-	if err != nil {
-		return "", err
-	}
+func createDockerfile(t *testing.T) (string, error) {
+	dir := t.TempDir()
```

Comment:
If we only want to use the `TempDir()` function, we could reduce the dependencies to `func createDockerfile(tempDir func() string) (string error)` and pass `t.TempDir` as parameter